docs/usecases/running_tests_on_cluster_not_deployed_by_ocsci.rst

```diff
@@ -0,0 +1,218 @@
+so that tests which are affected by known bugs will be skipped (see
+:doc:`/usage` for example of expected config file).
+
+Putting it all together, a full example of ocs-ci execution command would look
+like this:
+
+.. code-block:: console
+
+   (venv) $ run-ci --cluster-path=~/cluster-2020-10-02 \
+                   --ocsci-conf=conf/deployment/gcp/ipi_1az_rhcos_3m_3w.yaml \
+                   --cluster-name=mbukatov-2020-10-02 \
+                   --ocp-version=4.5 \
+                   --ocs-version=4.5 \
+                   --bugzilla \
+                   -m tier1 \
+                   --junit-xml=mbukatov.2020-10-02.tier1.xml \
+                   tests
+
+This example assumes that:
+
+- ``~/cluster-2020-10-02`` is a cluster dir, as created by
+  ``openshift-install ... --dir=~/cluster-2020-10-02`` command
+- Config file ``bugzilla.cfg`` (in ocs-ci directory or home directory) contains
+  reference and credentials to `Red Hat bugzilla`_
+- Config file ``conf/deployment/gcp/ipi_1az_rhcos_3m_3w.yaml`` describes both
+  platform the cluster is deployed on, and it's configuration - see details
+  in a section below to learn how to select or create it.
+- OCP version is ``4.5.1``, so we specify it as ``--ocp-version 4.5``.
+- OCS version is ``4.5.0-543.ci``, so we specify it as ``--ocs-version 4.5``
+  and don't use ``--live-deploy`` option.
+
+.. _`JUnit XML`: https://docs.pytest.org/en/stable/usage.html#creating-junitxml-format-files
+.. _`pytest cli tool`: https://docs.pytest.org/en/stable/usage.html
+.. _`bugzilla pytest plugin`: https://github.com/eanxgeek/pytest_marker_bugzilla
+
+How do I specify ocs-ci config with platform and deployment info?
+-----------------------------------------------------------------
+
+Check if platform where the cluster is deployed is available in
+`ocs-ci/conf/deployment`_ directory.
+
+If yes, go to the directory of the platform and check which config files
+matches your environment best. Then copy it into a new config file, open it
+and tweak it. Update at least the following values to match your cluster:
+
+- ``deployment_type``: either ``ipi`` or ``upi`` (this is important because
+  some tests cases are executed on particular type only)
+- ``region``: cloud region as specified during openshift-install driven
+  deployment (this is necessary to be correctly set for some MCG/NooBaa test
+  cases)
+- ``base_domain``: DNS zone as specified during openshift-install driven
+  deployment (this is necessary to be correctly set for some MCG/NooBaa test
+  cases)
+
+This is not a full list though, it's possible that further tweaks are necessary
+in special cases (such as LSO, external cluster, ...).
+
+When you are done with tweaking, pass this new config file to ``--ocsci-conf``
+option as explained in previous section.
+
+.. _`ocs-ci/conf/deployment`: https://github.com/red-hat-storage/ocs-ci/tree/master/conf/deployment
+
+Access to platform where the cluster is deployed (IPI only)
```

Comment:
tests that involve interaction with the platform API, will work with UPI as well in most cases.
For instance, most platform related tests will run over VMware UPI, which is suitable for that - https://github.com/red-hat-storage/ocs-ci/blob/709f7e3aea4d571a411405fc86dab31d41913241/ocs_ci/ocs/platform_nodes.py#L174

Reply:
Good point. I will drop "IPI only" from the title and maybe tweak the text.

Reply:
Addressed in b60e563